js/main.js

```diff
@@ -53,6 +53,16 @@ $(".search-close").on("click", function() {
   $('body').removeClass('search-on');
 });
 
+$("#search-input").on("input", function() {
+  var content = $(this).val();
+  $('#search-input').css({'background-image' : '', 'background-repeat' : '', 'background-size':'', 'background-position': ''});  
```

Comment:
Please avoid inline styles as it leads to code redundancy. Try playing around with opacity or jquery attribute fetch